sccp/figures/figureS2.py

```diff
@@ -0,0 +1,150 @@
+"""
+S2: Initial Attempt at Pf2 on the lupus data
+article: https://www.science.org/doi/10.1126/science.abf1970
+data: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE174188
+"""
+
+# GOAL: test Pf2 on lupus data, get visualizations for factor matrices
+
+# load functions/modules ----
+from .common import (
+    subplotLabel,
+    getSetup,
+    reorder_table
+)
+from ..parafac2 import parafac2_nd, Pf2X
+from ..imports.scRNA import tensorFy
+import pandas as pd
+import numpy as np
+import seaborn as sns
+import anndata
+from .common import subplotLabel, getSetup
+import matplotlib.pyplot as plt
+import scipy.cluster.hierarchy as sch
+import tensorly as tl
+
+
+####################################################################################
+## LOAD IN DATA -- CAN BE MOVED TO imports/scRNA 
+####################################################################################
+
+def load_lupus_data(third_axis= "ind_cov", n_rand = 100):
```

Comment:
This function can go into `imports/scRNA.py` so that it is defined once for all figures.